Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue2354.cs

```diff
@@ -8,12 +8,15 @@
 #if UITEST
 using NUnit.Framework;
 using Xamarin.UITest;
+using Xamarin.Forms.Core.UITests;
 #endif
 
 namespace Xamarin.Forms.Controls.Issues
 {
 #if UITEST
 	[NUnit.Framework.Category(Core.UITests.UITestCategories.Github5000)]
+	[NUnit.Framework.Category(Core.UITests.UITestCategories.Image)]
```

Comment:
Doesn't this one need the RequiresInternetConnection attribute?

Reply:
It's on the test itself

https://github.com/xamarin/Xamarin.Forms/blob/e4a86cef2fc24fcfce5db1f885e710c137e0bf65/Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue2354.cs#L127

I couldn't find a straightforward way to read the categories off the class other than scanning the assemblies but might as well just scan the assembly :-)

https://github.com/xamarin/Xamarin.Forms/pull/12790/files#diff-ad2af0a0a8e3e439c59c593a96c27eddcea548a938c7a47e02bd1ecc87bafa35R476